types/FavouriteList.d.ts

```diff
@@ -0,0 +1,3 @@
+declare namespace OIPF {
+    export type FavouriteList = OIPF.Collection<OIPF.Channel>;
```

Comment:
I'm not entirely sure why ChannelList uses type Channel but FavouriteList uses type OIPF.Channel

Reply:
Good spot - just a lack of consistency on my part. Will add the namespace as that is the convention.